examples/hybrid-compute/Quantinuum_hseries_hybrid_compute.ipynb

```diff
@@ -0,0 +1,1208 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Wasm Guidelines\n",
+    "\n",
+    "Some limitations exist on what kinds of functions or capabilities are enabled. \n",
+    "\n",
+    "1. Guidelines writing source code:\n",
+    "    * The `*.rs` or `*.c` source must contain an empty void method named `init`. This is to initialize the Wasm environment.\n",
+    "    * Examples of the types of functions that can be put into code are provided in the `c/src` or `rust/src` folders within each of the example folders.\n",
+    "    * The function to compile to Wasm must accept multiple integers and return exactly one integer.\n",
+    "    * Rust source code can use the Rust standard library.\n",
+    "    * C source code cannot use the C standard library, WASI is needed for this, which breaks the Wasm memory sandbox.\n",
+    "    * The following is not allowed:\n",
+    "        * Random number generation\n",
+    "        * File I/O\n",
+    "        * Exception Handling\n",
+    "1. Quantum programs can only call Wasm functions that accept multiple integers and return an integer. \n",
+    "    * For example, examine the call `x = wasm_func(a, b, c);` The `wasm_func` function can exist in QASM where `a`, `b`, `c`, and `x` are classical registers. \n",
+    "    * Such function calls have timing restrictions. Wasm calls in quantum programs have been run that took multiple milliseconds.\n",
+    "1. The state of a Wasm program persists between calls in the quantum programs within the limits of a job chunk window (~300 shots). \n",
+    "    * This means global variables can hold a mutable state that is modified between calls.\n",
+    "    * One can imagine using various global arrays. It's possible to have global pointers work with heap allocated data structures (for example using unsafe Rust features when compiling to Wasm).\n",
+    "1. Currently, the combined Quantum Web Assembly programs have to come in under 6 MB*."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To inject Wasm into a TKET program and enable processing on the H-Series devices or emulators, three components must be incorporated in the workflow.\n",
+    "\n",
+    "1. [pytket.wasm.WasmFileHandler](https://tket.quantinuum.com/api-docs/wasm.html#pytket.wasm.WasmFileHandler): Enables addition of the Wasm module to the TKET program as well as provides capability to verify logic within the Wasm module.\n",
```

Comment:
I am not sure if `verify logic within the Wasm module.` is the right way to express what is done. We are only checking the parameters types, return rypes and how many parameters there are. Nothing inside the function is checked. Do you think this is clear from this wording?

Reply:
Thanks